src/dom.js

```diff
@@ -16,6 +17,9 @@ import ComponentWidget from './component-widget'
 // particular library. For more information, see `./component-widget.js`.
 export default function dom (tag, properties, ...children) {
   if (typeof tag === 'function') {
+    if (!tag.prototype.render) {
+      tag = fromFunction(tag)
+    }
```

Comment:
Not 100% sure if this is the best method to detect if we're using a plain function instead of a component constructor